pkgs/development/ocaml-modules/xenstore/default.nix

```diff
@@ -9,21 +13,25 @@ buildDunePackage rec {
   minimalOCamlVersion = "4.08";
   duneVersion = "3";
```

Comment:
Can you remove this as well? This is the default behavior.

Reply:
Removed on [b952554](https://github.com/NixOS/nixpkgs/commit/b952554a81d842392bbfdca2ddf0164b9b0794c4).